# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nvidia-texture-tools/hotfix-no-install-libsquish.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nvidia-texture-tools-2.1.2/extern/libsquish-1.15/CMakeLists.txt.vanilla	2025-04-28 21:10:00.837323011 +0300
+++ nvidia-texture-tools-2.1.2/extern/libsquish-1.15/CMakeLists.txt	2025-04-28 21:10:27.588453908 +0300
@@ -108,10 +108,3 @@
         TARGET_LINK_LIBRARIES(squishpng squish ${PNG_LIBRARIES})
     ENDIF (PNG_FOUND)
 ENDIF (BUILD_SQUISH_EXTRA)
-
-INSTALL(
-    TARGETS squish
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib
-    PUBLIC_HEADER DESTINATION include
-    )
